flight/targets/brain/fw/pios_config.h

```diff
@@ -54,6 +54,8 @@
 #define PIOS_INCLUDE_FASTHEAP
 #define PIOS_INCLUDE_FRSKY_RSSI
 #define PIOS_INCLUDE_TBSVTXCONFIG
+#define PIOS_INCLUDE_DAC
```

Comment:
that's for telemetry... correct? I did not have a way to verify it, so I left it out. Happy to resubmit, but it would not be verified.